src/widgets/communication/ui/communication.test.tsx

```diff
@@ -0,0 +1,63 @@
+import { cleanup, screen } from '@testing-library/react';
+import { Communication } from './communication';
+import { renderWithRouter } from '@/shared/__tests__/utils';
+import { DISCORD_LINKS, communicationText } from 'data';
+
+describe('Communication section', () => {
+  it('should render "Communication" section texts correctly', async () => {
+    const firstCourse = Object.keys(DISCORD_LINKS)[0] as keyof typeof DISCORD_LINKS;
+
+    const {
+      title,
+      subTitle,
+      firstParagraphFirstHalf,
+      discordLink,
+    } = communicationText.en;
+
+    renderWithRouter(<Communication courseName={firstCourse} />);
```

Comment:
I would just use the string `'react'` or whatever for `courseName`

Reply:
I prefer to not rely on plain string. In this case, ```courseName``` is value from list of const. List and its value can change, but if the list persists, it's safe to use the first permitted value from this list.